packages/devtools/lib/src/framework/framework.dart

```diff
@@ -571,12 +673,60 @@ class ConnectDialog {
 
       // Hide the dialog
       hide();
+
```

Comment:
it can be a separate CL but it looks like what we have created now is a separate "connect" screen that prompts you to connect to a websocket or connect to a static file serving a snapshot of the app. Maybe add a TODO to refactor the connect dialog and the ImportMessage into their own screen? That could make some of this logic simpler although there are unanswered questions about if you should show the connect screen as its own full "tab".

Reply:
added a todo